ansible/inventory/env/group_vars/all.yml

```diff
@@ -658,3 +658,6 @@ kafka_urls: "{{groups['processing-cluster-kafka']|join(':9092,')}}:9092"
 kafka_topics_certificate_instruction: "{{env_name}}.coursebatch.certificate.request"
 cert_service_container_name: "{{env}}-e-credentials"
 cert_service_cloud_storage_type: "{{cert_service_cloud_storage_type}}"
+
+### Release-2.3.5 ###
+sunbird_portal_merge_auth_server_url: 'https://merge.dev.sunbirded.org/auth/'
```

Comment:
I think you can delete this variable as this is redundant
You can use this var in env file as the value {{sunbird_subdomain_keycloak_base_url}}